### PR TITLE
Use same origin instead of equals 

### PR DESCRIPTION
**Pull request by linnan-github**
***
<a href="https://pr-preview.s3.amazonaws.com/WICG/attribution-reporting-api/pull/475.html" title="Last updated on Jun 3, 2022, 2:46 PM UTC (fc98986)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/WICG/attribution-reporting-api/475/6d3b06c...fc98986.html" title="Last updated on Jun 3, 2022, 2:46 PM UTC (fc98986)">Diff</a>

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -723,7 +723,7 @@ To <dfn>process an attribution source</dfn> given an [=attribution source=] |sou
 
 1. Let |cache| be the user agent's [=attribution source cache=].
 1. [=list/Remove=] all entries in |cache| where all of the following are true:
-    * the entry's [=attribution source/attribution destination=] and |source|'s [=attribution source/attribution destination=] are equal.
+    * the entry's [=attribution source/attribution destination=] is [=same site=] with |source|'s [attribution source/attribution destination].
```
Comment (apasel422) [inline index.bs:726]: From the same site link:

> Two origins, A and B, are said to be same site...

But `attribution destination` is a site, so I'm not sure the operation is defined here, and `equals` may actually be correct.

Comment (linnan-github) [reply index.bs:726]: Yeah, that's right. Currently there are two places we use `same site` for a site, will replace them to `equals` as well.

File: index.bs [hunk 2]
```diff
@@ -101,7 +101,7 @@ name.
 <h3 id="monkeypatch-navigation">Navigation</h3>
 
 This section ensures that an [=attribution source=] associated with a navigation
-results in a top-level navigation whose final URL is [=same site=] with the
+results in a top-level navigation whose final URL equals the
```
Comment (apasel422) [inline index.bs:104]: I don't think this is exactly right. The URL itself might not equal the destination.

File: index.bs [hunk 3]
```diff
@@ -683,7 +683,7 @@ To <dfn>maybe process a navigation attribution source</dfn> given a <a spec="HTM
 1. If |browsingContext| is not a <a spec="html">top-level browsing context</a>, return.
 1. Let <var>attributionSource</var> be |navigationParams|'s [=navigation params/attribution source=].
 1. If |attributionSource| is null, return.
-1. If |attributionSource|'s [=attribution source/attribution destination=] is not [=same site=] to |navigationParams|'s
+1. If |attributionSource|'s [=attribution source/attribution destination=] does not equal |navigationParams|'s
```
Comment (apasel422) [inline index.bs:686]: Not sure this is right, either. The attribution destination is a site, but it's being compared to an origin here.

**Review by apasel422 — CHANGES_REQUESTED** [review]

File: index.bs (hunk 3 above)
Comment (linnan-github) [reply index.bs:686]: Yeah, probably better to get site from origin first and then compare the site. Will do this in a separate PR.

**Comment by apasel422** [comment]
LGTM, but please update the PR title

**Review by apasel422 — APPROVED** [review]

**Pull request merged**